src/main/java/com/solvd/webtest/pages/desktop/HomePage.java

```diff
@@ -52,6 +65,17 @@ public SearchResultPage search(String query, String category) {
         } catch (Exception e) {
             LOGGER.warn("Caught exception {}", e.toString());
         }
-        return new SearchResultPage(getDriver());
+        return initPage(getDriver(), SearchResultPageBase.class);
+    }
+
+    @Override
+    public CategoriesPageBase openCategoriesPage() {
+        if (categoriesMenu.isElementPresent(Duration.ofMillis(1000)))
```

Comment:
The default wait is long if the element is not present, its waits for 20 seconds. This element is not present on all version of site so it only checks for it if its present otherwise it would fail.